src/main/java/cofh/thermaldynamics/block/BlockDuct.java

```diff
@@ -282,16 +282,21 @@ public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World wo
 		if (target.subHit >= 14 && target.subHit < 20) {
 			TileGrid tileEntity = (TileGrid) world.getTileEntity(pos);
 			Attachment attachment = tileEntity.getAttachment(target.subHit - 14);
-			ItemStack pickBlock = attachment.getPickBlock();
```

Comment:
context: calling `getPickBlock()` on the attachment would cause a NPE directly after dismantling it with a crescent hammer